engines/dialogs.cpp

```diff
@@ -331,13 +331,17 @@ ConfigDialog::ConfigDialog() :
 	}
 
 	//
-	// The Achievements tab
+	// The Achievements & The Statistics tabs
 	//
 	Common::AchievementsInfo achievementsInfo = metaEngine->getAchievementsInfo(gameDomain);
 	if (achievementsInfo.descriptions.size() > 0) {
 		tab->addTab(_("Achievements"), "GlobalConfig_Achievements");
 		addAchievementsControls(tab, "GlobalConfig_Achievements.", achievementsInfo);
 	}
+	if (achievementsInfo.stats.size() > 0) {
```

Comment:
it's a style thing, and doesn't make any difference for most of our containers - but what you really wanna check is imo

`if (!achievementsInfo.stats.empty()) {`

for containers that doesn't cache its own size, it makes a difference (as I said, it's not issue here - more a style thing)

Reply:
Very good point, since it's a common engine cpp, we need to keep it neat =)
Fixes at  081ad8c